src/errorcodes.hpp

```diff
@@ -19,6 +19,9 @@
 
 #include <gdal.h>
 
+// warpbindings error codes should not intersect GDAL error codes
+#define ATTEMPTS_EXCEEDED 100
```

Comment:
Any suggestions about what is the better code number / name / the better place for it are welcome

Reply:
As long as this number does not conflict with any of GDAL's existing codes, it should be fine https://gdal.org/doxygen/cpl__error_8h.html .

Reply:
I'll add this link into the comment